modules/packages/LinearAlgebra.chpl

```diff
@@ -1308,6 +1308,47 @@ proc svd(A: [?Adom] ?t) throws
   return (u, s, vt);
 }
 
+/* 
+  Compute the approximate solution to ``A * x = b`` using Jacobi method,
+  iteration will stop when ``maxiter`` is reached or error is smaller than
+  ``tol``, whichever comes first. Return the number of iterations performed.
+  
+  .. note::
+    ``X`` is passed as a reference, meaning the initial solution guess can be
+    stored in ``X`` before calling the procedure, and the approximate solution 
+    will be stored in the same array.
+*/
+proc jacobi(A: [?Adom] ?eltType, ref X: [?Xdom] eltType, 
```

Comment:
Is there any particular reason for using uppercase X? Since it is a vector I'd expect it to be lower case.

Reply:
Originally it was small letter, but when I was making the last changes I recalled some syntax/aesthetic convention that made me capitalize it

Reply:
I can see some uppercase vectors in LinearAlgebra module as well. So it's fine.